transferui/src/main/java/com/hyperwallet/android/ui/transfer/TransferSourceWrapper.java

```diff
@@ -0,0 +1,73 @@
+package com.hyperwallet.android.ui.transfer;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.hyperwallet.android.model.transfermethod.TransferMethod;
+
+public class TransferSourceWrapper implements Parcelable {
```

Comment:
rename this to `TransferSource` as discussed in the design review, we only use `Wrapper` when two main Resource Entity are one object

Reply:
fixed

Reply:
fixed